pywbem_mock/_mainprovider.py

```diff
@@ -2557,14 +2558,15 @@ def _pull_response(self, req_type, EnumerationContext,
             eos = u'TRUE'
             rtn_objs_list = objs_list
             del self.enumeration_contexts[EnumerationContext]
-            context_id = ""
+            context = ""
```

Comment:
In the open method, we use context_id. Stay consistent.

Reply:
But the name in the named tuple is context so we are consistently inconsistent.  Since it is built from the method create_contextid() I went back to context_id for both of these methods.

DONE